tests/lowering/normalization/test_group_norm.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch_ttnn
+import pytest
+import ttnn
+
+from tests.utils import assert_with_pcc
+
+
+class GroupNormModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, embedding, num_groups, weight, bias):
+        return torch.nn.functional.group_norm(
+            embedding, num_groups=num_groups, weight=weight, bias=bias
+        )
+
+@pytest.mark.parametrize(
+    "num_groups, batch, sentence_length, embedding_dim",
+    [(3, 20, 6, 10)],
+)
+def test_group_norm(device, num_groups, batch, sentence_length, embedding_dim):
+    m = GroupNormModule()
+    input_shapes = [
+        (batch, sentence_length, embedding_dim),
+        (sentence_length),
+        (sentence_length),
+    ]
+    inputs = [torch.rand(shape, dtype=torch.bfloat16) for shape in input_shapes]
+    inputs.insert(1, num_groups)
+    result_before = m.forward(*inputs)
+    option = torch_ttnn.TorchTtnnOption(device=device)
+    option.gen_graphviz = True
+    # The compilation is lazy, so we need to run forward once to trigger the compilation
+    m = torch.compile(m, backend=torch_ttnn.backend, options=option)
+    result_after = m.forward(*inputs)
+    option._out_fx_graphs[0].print_tabular()
+
+    # Check the graph has be rewritten and contain ttnn ops
+    nodes = list(option._out_fx_graphs[0].nodes)
+    assert [node.target for node in nodes].count(ttnn.group_norm) == 1
```

Comment:
I made this test similar to `test_layer_norm.py`. This test fails because there is no `ttnn.group_norm` in the graph. Instead, there is `aten.native_group_norm.default`. Is this behavior expected?

```
----------------------------------------------------------------------------------- Captured stdout call ------------------------------------------------------------------------------------
opcode         name                       target                          args                                            kwargs
-------------  -------------------------  ------------------------------  ----------------------------------------------  --------
placeholder    arg0_1                     arg0_1                          ()                                              {}
placeholder    arg1_1                     arg1_1                          ()                                              {}
placeholder    arg2_1                     arg2_1                          ()                                              {}
call_function  native_group_norm_default  aten.native_group_norm.default  (arg0_1, arg1_1, arg2_1, 20, 6, 100, 3, 1e-05)  {}
call_function  getitem                    <built-in function getitem>     (native_group_norm_default, 0)                  {}
output         output                     output                          ((getitem,),)                                   {}
--------------------------------------------------------------------------------- Captured stdout teardown ----------------------------------------------------------------------------------
```

Reply:
@jdh8 , is this just a note or something actionable? 
ttnn.group_norm is available and if conversion is missing, you can call it if needed.

Reply:
I take notes for failed tests. There are still ops with higher priority than group norm, so I'm leaving it as is for now.